genet/core.py

```diff
@@ -2697,31 +2707,78 @@ def write_to_json(self, output_dir):
             self.schedule.write_to_json(output_dir)
         self.write_extras(output_dir)
 
-    def write_to_geojson(self, output_dir, epsg: str = None):
+    def write_spatial(
```

Comment:
If `write_spatial` is accessed from the `write_to_...` funcs lower down, maybe it should be a private function?

Reply:
got rid of the the `write_to_...` methods